Sources/VitaminUIKit/Components/Snackbar/README.md

```diff
@@ -0,0 +1,174 @@
+## Snackbar
+[Reference](https://www.decathlon.design/726f8c765/p/798580-snackbar/b/83feb7)
+
+Vitamin provides a `VitaminSnackbar` class to display a short message at the bottom of the screen
+
+You have two possibilities:
+- build your `VitaminSnackbar`, place it on your screen and make it appear
+- use the provided `UIViewController` extension to push a snackbar at the bottom of your ViewController's view
+
+### Basic properties
+A `VitaminSnackbar` has 3 basic properties
+- a `title` String, that should be short, and is mandatory
+- a `message`String, that should also be as short as possible, and is also mandatory
+- an optional `image` that will be displayed on the left hand sode of the snackbar
+
+If the image is set, the title and message will be placed on the right of the image, and aligned to left
+If not, title and content will be centered
+
+All these properties can be set in constructor, by passing them in a `VitaminSnackbar.BasicConfiguration` struct.
+They can also be set separately afterwards
+
+```swift
+// let's set properties in constructor
+let snackbar = VitaminSnackbar(
+    basicConfiguration: VitaminSnackbar.BasicConfiguration(
+        title: "A short title is best",
+        message: "A message should be short",
+        image: VitaminAssets.Placeholder.landscape.image
+    )
+)
+
+// or let's define an empty VitaminSnackbar and set its properties afterwards
+let snackbar2 = VitaminSnackbar()
+snackbar2.title = "Another short title is best",
+snackbar2.message = "A message should be shorter",
+snackbar2.image = VitaminAssets.Placeholder.landscape.image
```

Comment:
`snackbar2` is not really nice for a name 😅 I think that we can keep `snackbar`.
```suggestion
let snackbar = VitaminSnackbar()
snackbar.title = "Another short title is best",
snackbar.message = "A message should be shorter",
snackbar.image = VitaminAssets.Placeholder.landscape.image
```

Reply:
Since it is in the same block of code, I did not want to use twice the same variable name.
I changed it to `anotherSnackbar`